### DISCUSSION TITLE
Issue with Student Reverification

### DISCUSSION THREAD

**Discussion by HishaamA** [community/community#55316] [GitHub Education]
### Hi everyone,

I was previously a github student until yesrerday i got a message that i have to reverify. and when i tried to that it immediately says that its rejected. with the following message: Unfortunately, we weren't able to approve your educational discount request for [@HishaamA](https://github.com/HishaamA).
 What happened?
. Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

I used webcam to take a photo of my school ID. and I've used my school email address which was used before to verify.
I've tried 7 times to reverify, yet it doesn't work. 

Also I still have PRO on my profile and GitHub Copilot, but i can't access the developer pack

**Comment by samus-aran** [comment]
Hi @HishaamA 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
